core/baseline-core-v1.0-psd01.md

```diff
@@ -566,195 +568,642 @@ Below we list and define the components of each layer. The detailed requirements
+</figure>
+
+As depicted, identities and credentials are established outside of the context, and, therefore, scope of a BPI. Hence, it is incumbent on BPI participants -- Requesters, Providers and, if distinct, Operators -- to establish the trust context of acceptable identities and credentials for a BPI. This statement also applies to a network of BPIs which are to interoperate with one another.
+
+**[D8]** A unique identifier utilized within one or more BPIs SHOULD be linked to an (Legal) Entity accepted by BPI participants through a cryptographically signed, cryptographically verifiable, and cryptographically revocable credential based on the public keys associated with the unique identifier of the credential issuer.
+
+In the context of this document, a Legal Entity is an individual, organization or company that has legal rights and obligations.
+
+Note that credentials utilized within one ore more BPIs may be self-issued. The acceptance of self-issued credentials is up to the BPI participants that need to  rely on the claim(s) within a self-issued credential.
+
+**[R46]** The unique identifier of the (Legal) Entity MUST be the subject of the credential.
+
+**[R47]** The unique identifier of the issuer of the (Legal) Entity credential utilized in one or more BPIs MUST have a credential linking the unique identifier of the issuer to an (Legal) Entity accepted by the participants within aforementioned BPIs.
+
+**[D9]** A credential utilized within one or more BPIs SHOULD follow the W3C Verifiable Credential specification [[W3C VC](#####-W3C-VC)].
+
+**[R48]** A credential utilized within one or more BPIs MUST itself have a unique and resolvable identifier.
+
+Note, that the unique and resolvable identifier of a credential does not have to be associated with any cryptographic keys.
+
+**[R49]** If present, the status of a credential utilized within one or more BPIs MUST be discoverable by a party verifying the credential, the credential verifier.
+
+In the context of this document, a credential verifier is defined per the [W3C Verifiable Credential Standard](#####-W3C-VC).
+
+**[D10]** A credential utilized within one or more BPIs SHOULD be discoverable by a participant in said BPI(s).
+
+**[R50]** The presentation of a credential utilized within one or more BPIs MUST be cryptographically signed by the presenter of the credential, also known as the credential holder.
+
+See the [W3C Verifiable Credential Standard](#####-W3C-VC) for a definition of credential holder.
+
+**[R51]** If a credential holder is a BPI participant, the holder MUST have a unique identifier that has been established within the BPI context the holder operates in.
+
+As discussed in [Section 3.1](##31-Introduction-and-High-Level-Requirements), BPIs require either decentralized or strongly federated identifier/identity providers that have been agreed to by the participants in a BPI context of one or more BPIs.  
```

Comment:
remove `BPI` before `context` > `[...] in a context of one [...]`

Reply:
updated